pages/docs/guides/debounce.mdx

```diff
@@ -0,0 +1,72 @@
+export const description = 'Avoid unnecessary function invocations by de-duplicating events over a sliding time window. Ideal for preventing wasted work when a function might be triggered in quick succession.'
+
+# Debounce
+
+Debounce allows a function to delay execution until a series of events have stopped being received. This is useful for preventing wasted work when a function might be triggered in quick succession. Use cases for debounce include:
+
+* Prevent wasted work when handling events from user input that may change multiple times in a short time period
+* Delay processing of noisy webhook events until they have stopped being received
+* Ensuring that functions use the latest event within a series of updates (example: synchronization)
+
+## How to configure debounce
+
+<CodeGroup forceTabs>
+```ts
+export default inngest.createFunction(
+  {
+    id: "handle-webhook",
+    debounce: {
+      key: "event.data.account_id",
+      period: "5m",
+    },
+  },
+  { event: "intercom/company.updated" },
+  async ({ event, step }) => {
+    // This function will only be scheduled 5m after events have stopped being received with the same
```

Comment:
```suggestion
    // This function will only be scheduled 5 minutes after events are no longer received with the same
```

Reply:
Committing this, but it still feels weird - may be something to adjust later 🤔